mono/metadata/class-init.c

```diff
@@ -1533,6 +1533,14 @@ print_implemented_interfaces (MonoClass *klass)
 	}
 }
 
+static gboolean 
+mono_method_is_reabstracted(MonoClass *klass, guint16 flags)
```

Comment:
1. nit: space before `(`
2. `klass` is unused.   I think instead of taking `klass` and `flags`, this function should just get a `MonoMethod* method` argument and check `method->flags`
3. nit: static functions don't need a `mono_` prefix.

Reply:
I changed 1 and 3, but 2 I think that is not a good idea to receive the method because on count_virtual_methods, the methods aren't setup yet, and I think it is not setup every time because expends more time than getting only the flags. So I let the flags int this method, is this okay for you?